src/interactions/dragInteraction.ts

```diff
@@ -44,6 +44,10 @@ export module Interaction {
     }
 
     private _startDrag(p: Point, e: UIEvent) {
+      var button = (<MouseEvent> e).button;
+      if (button != null && button !== 0) {
+        return;
+      }
```

Comment:
Wouldn't a more semantical check be...

if (e instanceof MouseEvent && button !== 0) {
  return;
}

?